packages/subgraph/schema.graphql

```diff
@@ -90,7 +90,7 @@ type Dao @entity {
   deposits: [VaultDeposit!]! @derivedFrom(field: "dao")
   withdraws: [VaultWithdraw!]! @derivedFrom(field: "dao")
   balances: [Balance!] @derivedFrom(field: "dao")
-  roles: [Role!]! @derivedFrom(field: "dao")
+  contractPermissionIDs: [ContractPermissionID!]! @derivedFrom(field: "dao")
```

Comment:
```suggestion
  permissionIds: [ContractPermissionID!]! @derivedFrom(field: "dao")
```

Reply:
Duplicate of https://github.com/aragon/core/pull/62#discussion_r928966168. Closing this conversation